source/kann-metadata.json

```diff
@@ -0,0 +1,182 @@
+        "category": "Data"
+    },
+    {
+        "name": "MaxWithConstant",
+        "category": "Transform"
+    },
+    {
+        "name": "ConvolutionCV2",
+        "category": "Layer"
+    },
+    {
+        "name": "Reciprocal",
+        "category": "Layer"
+    },
+    {
+        "name": "ReLU",
+        "category": "Activation"
+    },
+    {
+        "name": "Resize",
+        "category": "Shape"
+    },
+    {
+        "name": "ResizeNearestAsymFloor",
+        "category": "Shape"
+    },
+    {
+        "name": "MulElementwise",
+        "category": "Transform"
+    },
+    {
+        "name": "Copy",
+        "category": "Transform"
+    },
+    {
+        "name": "Exp",
+        "category": "Activation"
+    },
+    {
+        "name": "Mish",
+        "category": "Activation"
+    },
+    {
+        "name": "Stddev",
+        "category": "Normalization"
+    },
+    {
+        "name": "Division",
+        "category": "Layer"
+    },
+    {
+        "name": "MatMul",
+        "category": "Transform"
+    },
+    {
+        "name": "AvgPooling",
+        "category": "Pool"
+    },
+    {
+        "name": "HardSigmoid",
+        "category": "Activation"
+    },
+    {
+        "name": "MaxElementwise",
+        "category": "Transform"
+    },
+    {
+        "name": "SeLU",
+        "category": "Activation"
+    },
+    {
+        "name": "LRN",
+        "category": "Activation"
+    },
+    {
+        "name": "MulWithConstant",
+        "category": "Transform"
+    },
+    {
+        "name": "Abs",
+        "category": "Transform"
+    },
+    {
+        "name": "Invert",
+        "category": "Transform"
+    },
+    {
+        "name": "Sqrt",
+        "category": "Transform"
+    },
+    {
+        "name": "MinElementwise",
+        "category": "Transform"
+    },
+    {
+        "name": "GeLU",
+        "category": "Activation"
+    },
+    {
+        "name": "Input",
+        "category": "Data"
+    },
+    {
+        "name": "Logistic",
+        "category": "Activation"
+    },
+    {
+        "name": "Reshape",
+        "category": "Shape"
+    },
+    {
+        "name": "AddElementwise",
+        "category": "Transform"
+    },
+    {
+        "name": "Saturate",
+        "category": "Transform"
+    },
+    {
+        "name": "Tanh",
+        "category": "Activation"
+    },
+    {
+        "name": "ScalingVector",
+        "category": "Transform"
+    },
+    {
+        "name": "MaxPooling",
+        "category": "Pool"
+    },
+    {
+        "name": "Convert",
+        "category": "Transform"
+    },
+    {
+        "name": "DepthwiseConvolution",
+        "category": "Layer"
+    },
+    {
+        "name": "Softmax",
+        "category": "Activation"
+    },
+    {
+        "name": "MinWithConstant",
+        "category": "Transform"
+    },
+    {
+        "name": "WriteValue",
+        "category": "Data"
+    },
+    {
+        "name": "SiLU",
+        "category": "Activation"
+    },
+    {
+        "name": "ImageSMEM",
```

Comment:
As explained, KaNN handles tensors and layers differently than other implementations, so it would be valuable to keep model-specific operators to help users visually understand its structure.

Reply:
Any operators which are not standard across formats. For example, "ImageSMEM" does not seem to be documented in online search. Its function or how it relates to other similar operators in other frameworks is not clear. Start by leaving it blank and create a follow-up issue in a KaNN repository to do such an investigation across formats and update categories for all similar operators in all formats.

Reply:
This would be `ImageSMEM` and `ImageDDR` then. However in relation with the global answer, those two operators are not actual layers but are necessary for Netron to visualize KaNN's unique tensor memory management at a lower level and will not be encountered in any other frameworks. They will be removed if it's non negociable, but doing so will be a significant loss of information for understanding our models.

Reply:
Ok, this makes sense. Those operators are removed from metadata and we'll investigate how to categorize them properly for `ImageSMEM` and `ImageDDR` really need to be displayed distinctly.

Reply:
"LRN" is category "Normalization".